parsers/c/c_lexer.mll

```diff
@@ -84,6 +84,8 @@ let keywords: (string * Tokens.token) list = [
     "__builtin_types_compatible_p", BUILTIN_TYPES_COMPATIBLE_P;
     "__builtin_choose_expr", BUILTIN_CHOOSE_EXPR;
 
+    "==>", CN_IMPLIES;
```

Comment:
`"implies" , CN_IMPLIES;` Also this should be in `cn_keywords`, not in `keywords, which is for C keywords (see [this comment](https://github.com/rems-project/cerberus/pull/352#discussion_r1654956556).